Classes/Controllers/PBGitCommitController.h

```diff
@@ -9,43 +9,16 @@
 #import <Cocoa/Cocoa.h>
 #import "PBViewController.h"
 
-@class PBGitIndexController;
-@class PBIconAndTextCell;
-@class PBWebChangesController;
 @class PBGitIndex;
-@class PBNiceSplitView;
-@class PBCommitMessageView;
 
-@interface PBGitCommitController : PBViewController {
-	// This might have to transfer over to the PBGitRepository
-	// object sometime
-	PBGitIndex *index;
-
-	IBOutlet PBCommitMessageView *commitMessageView;
-
-	BOOL stashKeepIndex;
-
-	IBOutlet NSArrayController *unstagedFilesController;
-	IBOutlet NSArrayController *cachedFilesController;
-	IBOutlet NSArrayController *trackedFilesController;
-
-	IBOutlet NSTabView *controlsTabView;
-	IBOutlet NSButton *commitButton;
-	IBOutlet NSButton *stashButton;
-
-	IBOutlet PBGitIndexController *indexController;
-	IBOutlet PBWebChangesController *webController;
-	IBOutlet PBNiceSplitView *commitSplitView;
-}
-
-@property(readonly) PBGitIndex *index;
-@property(assign) BOOL stashKeepIndex;
+@interface PBGitCommitController : PBViewController
 
 - (IBAction) refresh:(id) sender;
```

Comment:
I can't say any more. Given that the primary caller of IBActions is NIB files, and those don't really care about public/private (since everything is really public).

I understand your view, but I can give a counter-example : most of `PBGitCommitController` IBActions now depend on having the correct sender passed to them (IOW, a menu item, which ~has its menu delegate set to~ I can't even explain anymore, look around `-validateMenuItem:` 😞). I'm not really satisfied with that, but for the time-being it works (and I went through so much hidden hoops that I felt this was the best solution). I should have left a FIXME in here.